ui/src/main/webapp/home/actions.js

```diff
@@ -0,0 +1,37 @@
+import { poll } from 'redux-polling'
+import { get, post } from 'redux-fetch'
+
+// actions
+
+export const setConfigs = (id, value) => ({type: 'HOME/SET_CONFIGS', id, value})
+export const beginDelete = (id, servicePid) => ({type: 'HOME/BEGIN_SUBMITTING_CONFIGS', id, servicePid})
```

Comment:
It's not. Originally I was going to keep the submitting state alongside the `home/configs` in the state tree and have it all in the same reducer. Removed this as well as the `servicePid` destructure in the configs reducer for the same reason.